setup.py

```diff
@@ -31,7 +31,8 @@ def find_version(*file_paths):
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 test_deps = [
     "pytest==4.5",
```

Comment:
still no luck with newest pytest/pytest-cov?

Reply:
i tried again with various combos of pytest/pytest-cov; can't figure it out

Reply:
Yeah, i had a try again at this, ... and gave up :(